.github/src/signin.css

```
@@ -0,0 +1,162 @@
+body {
+  display: flex;
+  background-color: #F0F6FF;
+  width: 1440px;
+  padding: 238px 0px 252px 0px;
```

```suggestion
justify-content: center;
```
이부분은 아마 중간 정렬하기 위해서 쓴거같은데 padding보다는 justifycontent로 정렬하시는게 좋아요:) 왜냐하면 반응형으로 개발할떄 이런식으로 스태틱하게 값을 적용하면 반응형 개발이 힘들어 지기 때문이에요